src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -495,19 +500,22 @@ int xccdf_policy_rule_result_remediate(struct xccdf_policy *policy, struct xccdf
 
 	if (fix == NULL) {
 		fix = _find_suitable_fix(policy, rr);
-		if (fix == NULL)
-			// We may want to append xccdf:message about missing fix.
```

Comment:
The comment should be kept because it hasn't been addressed. The comment is about adding a message to the XCCDF results which is the big XML file generated using `--results` option.

Reply:
yes my bad it's because at this time I had not understand the diff between DI and xccdf:message :(

Reply:
That's fine.

Reply:
In the last PR i've a little modify the comment since it's now reported ... i think now it can be even removed but I've prefer to keep too much comments than not enough.